### PR TITLE
Prepare rollout isOnV2

### PR DESCRIPTION
**Pull request by PopDaph**
Migrate Dust workspace on v2 (and local environments).
We hardcode the sids of the rolled out workspaces on `V2_ROLLED_OUT_WORKSPACES`.

### CODE REVIEW

File: front/components/sparkle/navigation.tsx [hunk 1]
```diff
@@ -61,29 +62,33 @@ export const topNavigation = ({
...
+  const isOnV2 = isOnAssistantV2(owner);
   const isDust = isDevelopmentOrDustWorkspace(owner);
-  const nav: SparkleAppLayoutNavigation[] = [
-    {
-      id: "assistant",
+  const nav: SparkleAppLayoutNavigation[] = [];
+
+  if (isOnV2) {
+    nav.push({
+      id: "assistants",
```
Comment (spolu) [inline front/components/sparkle/navigation.tsx:71]: typo?

**Review by spolu — DISMISSED** [review]
Let's point settings to assistants manager once rolled out instead of data sources?

File: front/components/sparkle/navigation.tsx (hunk 1 above)
Comment (PopDaph) [reply front/components/sparkle/navigation.tsx]: I needed different ids for V1 and V2 when we were displaying both but indeed now we don't need it anymore, thanks for catching!

**Review by spolu — APPROVED** [review]

**Comment by PopDaph** [comment]
> Let's point settings to assistants manager once rolled out instead of data sources?
Done! 🙏🏻

**Pull request merged**